ledger-core/v2/application/testwalletapi/statemachine/statemachine.go

```diff
@@ -58,7 +58,12 @@ func (s *SMTestAPICall) GetStateMachineDeclaration() smachine.StateMachineDeclar
 	return testAPICallSMDeclarationInstance
 }
 
+func (s *SMTestAPICall) defaultMigration(ctx smachine.MigrationContext) smachine.StateUpdate {
+	return ctx.Jump(s.stepSendRequest)
```

Comment:
this is wrong implementation. it generates completely new outgoing request and this is not what expected from resend.

Reply:
good point, fixed